src/core/queryClient.ts

```diff
@@ -123,6 +123,31 @@ export class QueryClient {
       .setData(updater, options)
   }
 
+  setQueriesData<TData>(
```

Comment:
since we are not creating queries like with `setQueryData`, because we only update matching keys, would `updateQueries` or `updateQueriesData` be a better name?

this was also mentioned by @tannerlinsley here: https://github.com/tannerlinsley/react-query/issues/135#issuecomment-589710815

but I don't think it made it?

Reply:
I think the current name is fine as long as people realize that it will not create queries.